downloader/src/main/kotlin/VcsHost.kt

```diff
@@ -385,7 +385,7 @@ private fun projectUrlToUserOrOrgAndProject(projectUrl: URI): Pair<String, Strin
 
```

Comment:
Commit message: It would be nice to give a reason why the suffix is removed.

Reply:
Oh, I though 'Strip any ".git" suffix from the project name' already makes clear that ".git" should not be regarded as a part of the project name, but I'll see what I can do.

Reply:
Thanks for bearing with my limited knowledge ;-)